### PR TITLE
Migrates 'test-for-missing-migrations' from Travis CI to GH WF

### LINKED ISSUE #1553: Travis CI -> GitHub workflow: test-for-missing-migrations
Kiwi TCMS needs help migrating certain test jobs from Travis CI into GitHub workflows. In particular the `test-for-missing-migrations` job.

**Definition of Done**:
- "test-for-missing-migrations" job has been removed from Travis CI
- "sanity / test-for-missing-migrations" job is enabled for GitHub workflows. See `.github/workflows/lint.yml` and create a `sanity.yml`
- The new job reports its result on the pull request

*This issue is part of Kiwi TCMS open source bounty program. For more information see the link(s) in [bounty-program milestone](https://github.com/kiwitcms/Kiwi/milestone/4)*

### PR DESCRIPTION
**Pull request by brymut**
Resolves #1553. Migrates 'test-for-missing-migrations' from Travis CI to GH WF

### CODE REVIEW

File: tcms/settings/test/__init__.py [hunk 1]
```diff
@@ -32,3 +32,9 @@
 # for running localized tests, see f74c3c1
 # See https://code.djangoproject.com/ticket/29713
 LANGUAGE_CODE = os.environ.get('LANG', 'en-us').lower().replace('_', '-').split('.')[0]
+print("================================> LANGUAGE_CODE", LANGUAGE_CODE)
```
Comment (atodorov) [inline tcms/settings/test/__init__.py:35]: export LANG=en-us before running in GH WF, see other jobs

File: .github/workflows/sanity.yml [hunk 2]
```diff
@@ -42,3 +42,25 @@ jobs:
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: test-for-missing-migrations
+      run: |
+        sudo apt-get install libkrb5-dev
+        export TEST_DB="SQLite"
+        export REQUIREMENTS_TXT="requirements/base.txt"
+        pip install -r $REQUIREMENTS_TXT
```
Comment (atodorov) [inline .github/workflows/sanity.yml:63]: only devel.txt will do, it includes the base requirements

File: .github/workflows/sanity.yml [hunk 3]
```diff
@@ -42,3 +42,25 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: test-for-missing-migrations
+      run: |
+        sudo apt-get install libkrb5-dev
+        export TEST_DB="SQLite"
```
Comment (atodorov) [inline .github/workflows/sanity.yml:61]: not needed

File: .github/workflows/sanity.yml [hunk 4]
```diff
@@ -42,3 +42,25 @@ jobs:
...
+        export REQUIREMENTS_TXT="requirements/base.txt"
+        pip install -r $REQUIREMENTS_TXT
+        pip install -r requirements/devel.txt
+        pushd tcms/ && npm install && popd
+        make test_for_missing_migrations
```
Comment (atodorov) [inline .github/workflows/sanity.yml:66]: add 1 empty line for readability before make

File: Makefile [hunk 5]
```diff
@@ -40,6 +40,7 @@ test:
 # https://stackoverflow.com/questions/54177838/
 .PHONY: test_for_missing_migrations
 test_for_missing_migrations:
+	echo $(DJANGO_SETTINGS_MODULE)
```
Comment (atodorov) [inline Makefile:43]: Not needed

**Comment by brymut** [comment]
Sorry, I'm still debugging, having issues with the Language code on GH,

**Pull request merged**